btr-web/btr-common-components/nuxt.config.ts

```diff
@@ -1,4 +1,5 @@
 // https://nuxt.com/docs/api/configuration/nuxt-config
+// @ts-ignore
```

Comment:
can you comment why we want the ts-ignore here and below?

Reply:
yeah, used this one automatically, should not be here, will remove them and will remove unused entries in config